### DISCUSSION TITLE
Add-on installation failed

### DISCUSSION THREAD

**Discussion by gerarddolan** [argenos/zotero-mdnotes#177] [Q&A] [answered]
I'm receiving the following error while trying to add MD Notes

Add-on installation failed
"The add-on "Mdnotes for Zotero" could not be installed. It may be incompatible with this version of Zotero.

Zotero 6.0.4

Any help would be appreciated.

**Comment by Antaur-Zhang** [comment, marked as answer]
You can see the discussion here.
Installation files for zotero 6 can be downloaded here
https://github.com/argenos/zotero-mdnotes/releases/tag/0.2.0-alpha5
I tried it and it can be installed successfully
